idaes/core/tests/test_components.py

```diff
@@ -419,55 +429,47 @@ def get_metadata(self):
             return m.meta_object
         m.get_metadata = types.MethodType(get_metadata, m)
 
-        m.comp = Ion()
-
-        return m
+        with pytest.raises(
+                NotImplementedError,
+                match="comp The IonData component class is intended as a base "
```

Comment:
Is "comp" supposed to be here or was it left here unintentionally?

Reply:
I am guessing it links to line 437 but not sure since I've never used this functionality.

Reply:
This is meant to be here - this is checking that the expected error message is returned, and `comp` is the name of the `Component` where the issue arose.